Source/WebKit/Platform/cocoa/MediaCapability.mm

```diff
@@ -37,21 +38,11 @@
 
 using WebCore::RegistrableDomain;
 
-static RetainPtr<_SECapability> createPlatformCapability(const URL& url)
-{
-#if USE(EXTENSIONKIT)
-    if (_SECapability *capability = [get_SECapabilityClass() mediaWithWebsite:RegistrableDomain(url).string()])
-        return capability;
-#endif
-
-    UNUSED_PARAM(url);
-    return nil;
-}
-
 MediaCapability::MediaCapability(URL url)
     : m_url { WTFMove(url) }
-    , m_platformCapability { createPlatformCapability(m_url) }
 {
+    m_mediaEnvironment = adoptNS([[BEMediaEnvironment alloc] initWithWebPageURL:m_url]);
```

Comment:
Why isn't this part of the initializer list?

Reply:
I did not add it to the initializer list, since it caused a style error. Would you prefer moving it into the initializer list?

Reply:
Yes, this should be in the initializer list.

Reply:
I agree, will move!